examples/Textual-Responses.ipynb

```diff
@@ -0,0 +1,337 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EiGsBj4xlMzY"
+      },
+      "source": [
+        "Welcome to this tutorial! We will look at how to apply aggregation methods to a text response, just imagine: you have received different text answers to the same transcription of the text and you need to somehow solve this problem and get the correct answer. Amazing!"
```

Comment:
Please add a clear description of the task. Don't use "imagine": we have a clearly defined task in CrowdSpeech, please use it